webapp/src/routes/Home.js

```diff
@@ -0,0 +1,220 @@
+/* eslint camelcase: 0 */
```

Comment:
Why add the home file again?

we moved this file to `webapp/src/routes/Home/index.js`

please check it!